cl/recap_rss/tasks.py

```diff
@@ -330,7 +331,7 @@ def merge_rss_feed_contents(self, feed_data, court_pk, metadata_only=False):
     d_pks_to_alert = []
     for docket in feed_data:
         item_hash = hash_item(docket)
-        if is_cached(item_hash):
+        if async_to_sync(is_cached)(item_hash):
```

Comment:
I'm new to async. Why call `async_to_sync` here?

Reply:
This is because `merge_rss_feed_contents` is a non-async function while `is_cached` is an async function, so we have to adapt here when calling async code from non-async.

Reply:
Right, thanks.